test/query_users.js

```diff
@@ -66,13 +66,17 @@ describe('Query Users', function() {
 				name: 'Roxanne',
 			},
 		]);
-		const response = await serverClient.queryUsers({
-			unique,
-			$or: [
-				{ name: { $autocomplete: 'ro' } },
-				{ username: { $autocomplete: 'ro' } },
-			],
-		});
+		const response = await serverClient.queryUsers(
+			{
+				unique,
+				$or: [
+					{ name: { $autocomplete: 'ro' } },
+					{ username: { $autocomplete: 'ro' } },
+				],
+			},
+			{ name: 1 },
```

Comment:
That's good but default order, creation time, should also have worked in the first place.

Any explanation why it doesn't, precision lose in js ?

Reply:
Even though the updateUsers() accepts a list of users, the api call sends a map. I think this makes the creation order non-deterministic (edited)

Reply:
We could do separate calls and put delay between calls to ensure creation time.

However, it would make tests slower so adding another sort order is good.